test-data/local/batch_sim/jobs/square_params.py

```diff
@@ -12,6 +14,7 @@ def copy_file(input_file, output_file):
             sq = data[key] * data[key]
             f.write("%g\n" % sq)
 
+    time.sleep(random.randint(0,4))
```

Comment:
>  Updated test job with some randomness. 

Perhaps you can add a comment as to why we sleep a random int?  Why not just `sleep(1)` if that's necessary?